src/NuGet.Indexing/ILoader.cs

```diff
@@ -2,11 +2,16 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Newtonsoft.Json;
+using System.Threading.Tasks;
 
 namespace NuGet.Indexing
 {
     public interface ILoader
     {
         JsonReader GetReader(string name);
+
+        AzureDirectorySynchronizer GetSynchronizer();
```

Comment:
I think the reason I put it here was because I thought it somewhat made sense to have it live with the [`StorageLoader`](https://github.com/NuGet/NuGet.Services.Metadata/pull/91/files#diff-abd3dac6b87987c81b2d39954087afdaR20). Would you prefer it to be in its own class?